snooty/types.py

```diff
@@ -311,6 +312,21 @@ def render_constants(self) -> Tuple["ProjectConfig", List[Diagnostic]]:
 
     def read(self, path: Path) -> Tuple[str, List[Diagnostic]]:
         text = path.read_text(encoding="utf-8")
+        PAT_GIT_MARKER = re.compile(r"^<<<<<<< .*?^=======\n.*?^>>>>>>>", re.M | re.S)
```

Comment:
This needs to be lifted out as a global constant; we don't want to be compiling a regexp every time we load a file